### PR TITLE
improve

### PR DESCRIPTION
**Pull request by tien7668**

### CODE REVIEW

**Review by piavgh — APPROVED** [review]

File: src/components/swapv2/styleds.tsx [hunk 1]
```diff
@@ -219,7 +219,7 @@ export function SwapCallbackError({ error }: { error: string }) {
     <SwapCallbackErrorInner>
       <Alert style={{ marginBottom: 'auto' }} />
       <AutoColumn style={{ flexBasis: '100%', margin: '10px 0 auto 8px' }}>
-        <Text fontSize="16px" fontWeight="500" color={theme.red}>
+        <Text fontSize="16px" fontWeight="500" color={theme.red} style={{ lineHeight: '24px' }}>
```
Comment (viet-nv) [inline src/components/swapv2/styleds.tsx:222]: Text component also has lineHeight props, we dont need inline style here

File: src/components/swapv2/styleds.tsx [hunk 2]
```diff
@@ -231,7 +231,7 @@ export function SwapCallbackError({ error }: { error: string }) {
           Show more details
         </Text>
         {showDetail && (
-          <Text color={theme.text1} fontSize="10px" style={{ margin: '10px 0 4px 0' }}>
+          <Text color={theme.text1} fontSize="10px" style={{ margin: '10px 0 4px 0', lineHeight: '16px' }}>
```
Comment (viet-nv) [inline src/components/swapv2/styleds.tsx:234]: Same as above, magin also is one of props of Text

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/components/swapv2/styleds.tsx (hunk 1 above)
Comment (tien7668) [reply src/components/swapv2/styleds.tsx]: agree

File: src/components/swapv2/styleds.tsx (hunk 2 above)
Comment (tien7668) [reply src/components/swapv2/styleds.tsx]: agree

**Pull request merged**